apollo-integration/build.gradle.kts

```diff
@@ -11,6 +11,7 @@ dependencies {
   add("implementation", "com.apollographql.apollo:apollo-rx3-support")
   add("implementation", "com.apollographql.apollo:apollo-coroutines-support")
   add("implementation", "com.apollographql.apollo:apollo-http-cache")
+  add("implementation", "com.apollographql.apollo:apollo-normalized-cache-sqlite")
```

Comment:
It's nice to be able to test the SqliteCache without Android 😃

Reply:
Definitely. It's a big win.